### DISCUSSION TITLE
Esta bien como va quedando mi Wireframe?

### DISCUSSION THREAD

**Discussion by ChanLuis** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#174] [Q&A]
![image](https://user-images.githubusercontent.com/99062113/156203379-615b3839-b34c-47ff-8316-c9bdc57c8820.png)

**Comment by davidtoString** [comment]
¡Bien! 
Al ser un wireframe, se entiende que es de baja calidad. Solo te recomendaría dejar más espacio en los márgenes, pues eso le da una apariencia más limpia.

**Reply by ChanLuis** [reply]
Gracias por el consejo

**Comment by doguedogue** [comment]
Se ve genial, sigue asi! 🚀

**Reply by ChanLuis** [reply]
Gracias por la ayuda
